packages/strapi/lib/middlewares/public/index.js

```diff
@@ -114,5 +133,40 @@ module.exports = strapi => {
         })
       );
     },
+
+    template: async (data, isInitialised) => {
+      // Allowed expressions to avoid data leaking.
+      const allowedExpression = [
```

Comment:
You can simply create an object by picking the values out of the strapi global. That would be more readable.